main.go

```diff
@@ -286,10 +304,19 @@ func main() {
 	}
 
 	stepconf.Print(config)
-
 	fmt.Println()
 
-	if err := mainE(config); err != nil {
+	//
+	// app_path_pattern is required, and the apk_path_pattern is deprecated
+	if config.APKPathPattern != "" {
+		log.Warnf("apk_path_pattern input is **DEPRECATED** - use the app_path_pattern input instead.")
+	}
+	if config.APKPathPattern != "" && config.AppPathPattern != "" {
```

Comment:
pls have a look at this: https://github.com/bitrise-steplib/steps-gradle-runner/pull/65/files#r285005653

Reply:
Fixed in:
https://github.com/bitrise-steplib/bitrise-step-android-build/pull/11/commits/9781c7142a0c1318f83e33e1126843ac57992766